spec/requests/flavors_spec.rb

```diff
@@ -0,0 +1,163 @@
+RSpec.describe 'Flavors API' do
+  let(:ems) { FactoryGirl.create(:ems_cloud) }
+  let(:flavor) { FactoryGirl.create(:flavor_openstack, :ext_management_system => ems) }
+  let(:flavor_2) { FactoryGirl.create(:flavor_openstack, :ext_management_system => ems) }
+
+  describe 'GET/api/flavors' do
+    it 'lists all the flavor bases with an appropriate role' do
```

Comment:
It's important to create a unified language both when writing tests and creating an API. Are they flavors or flavor bases?